### DISCUSSION TITLE
Hyprland 0.34 currently works on Debian Trixie [AMD graphics card]

### DISCUSSION THREAD

**Discussion by shh0** [JaKooLit/Hyprland-Dots#150] [General]
Hi,
just wanted to note, that Debian Trixie sources now work for Hyperland 0.34. Libdrm version is the needed one. So there's no need to stay at v0.32.3.
- add the package libtomlplusplus-dev
- in install-scripts/hyprland.sh : remove the specific version from the 'git clone' line.

I got some issues with waybar switching between light/dark theme. But, the rest seems to work nicely.
However, im using an AMD graphics card, there may be issues with other cards.
Best regards...

**Comment by JaKooLit** [comment]
thanks... Ill take care this on sunday

been busy with work

**Comment by JaKooLit** [comment]
going back to this,

Im slowing down in Debian-Hyprland project... its a pain that Debian cant keep up with newer packages
